packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -108,11 +108,11 @@ export interface TableProps extends CommonProps {
 
   // events
 
-  onSort?: (e?: Event) => void;
-  onGroup?: (e?: Event) => void;
-  onRowSelected?: (e?: Event) => any;
-  onRowExpandChange?: (e?: Event) => any;
-  onColumnsReordered?: (e?: Event) => void;
+  onSort?: (e: CustomEvent<{column: unknown; sortDirection: string}>) => void;
+  onGroupBy?: (e: CustomEvent<{column: unknown; isGrouped: boolean}>) => void;
```

Comment:
This would be a breaking change. Also the `prop` name and the affiliated function has to be changed then.

Reply:
this was not intended - reverted.